pyanaconda/modules/payload/live/live_image.py

```diff
@@ -26,7 +26,8 @@
 from pyanaconda.core.constants import INSTALL_TREE
 from pyanaconda.modules.common.constants.objects import LIVE_IMAGE_HANDLER
 from pyanaconda.modules.common.errors.payload import SourceSetupError
-from pyanaconda.modules.payload.handler_base import PayloadHandlerBase
+from pyanaconda.modules.payload.shared.handler_base import PayloadHandlerBase
```

Comment:
I don't like the name `shared`. In Anaconda, we use `base`, `utils`, `lib` or `common`. Please, choose one of the existing names.

Reply:
I was thinking about common in the beginning but it doesn't sound to me like something which is "shared". The base sounds better to me.
I'll go with base here.

Reply:
Fixed.